.gitignore

```diff
@@ -1,5 +1,6 @@
 nimcache/
 nimblecache/
-htmldocs/
+docs/dimscord.idx
+docs/dimscord/*.idx
```

Comment:
Aren't these needed for searching and probably procedure lists?

Reply:
They are only needed for the buildIndex command, after that it uses the theindex.html to search

Reply:
Ah alright.